.eslintrc.js

```diff
@@ -19,7 +19,9 @@ module.exports = {
   env: {
     browser: true,
   },
-  rules: {},
+  rules: {
+    'prettier/prettier': 'off',
```

Comment:
we typically keep these on, we don't agree with all formatting rules per se but it's easier to follow the community standard than to come up with your own :). You can easily fix most errors with `npm run lint:fix`

Reply:
total oversight, I don't think I even meant to disable prettier, not even sure when I did that. let me change that real quick